lib/SaferpayJson/Response/Container/ThreeDs.php

```diff
@@ -32,10 +32,17 @@ final class ThreeDs
 
     /**
      * @var string|null
-     * @SerializedName("VerificationValue")
+     * @SerializedName("Version")
      * @Type("string")
      */
-    private $verificationValue;
```

Comment:
Note to myself: This is a BC break and will therefore result in a new major version.